src/core_modules/capture-core/components/MapCoordinates/MapCoordinatesModal.js

```diff
@@ -0,0 +1,153 @@
+            // $FlowFixMe
+            setPosition(newPosition);
+        }
+    };
+
+    const onMapPolygonCreated = (e: any) => {
+        const polygonCoordinates = e.layer.toGeoJSON().geometry.coordinates;
+        setCoordinates(polygonCoordinates);
+    };
+
+    const onMapPolygonEdited = (e: any) => {
+        const polygonCoordinates = e.layers.getLayers()[0].toGeoJSON().geometry.coordinates;
+        setCoordinates(polygonCoordinates);
+    };
+    const onMapPolygonDelete = () => {
+        setCoordinates(null);
+    };
+
+    const renderMap = () => (<Map
+        center={center}
+        zoom={13}
+        ref={(ref) => {
+            if (ref?.leafletElement) {
+                setTimeout(() => { ref.leafletElement.invalidateSize(); }, 250);
+            }
+        }}
+        className={classes.map}
+        onClick={onHandleMapClicked}
+    >
+        <TileLayer
+            url="http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+            attribution="&copy; <a href=&quot;http://osm.org/copyright&quot;>OpenStreetMap</a> contributors"
+        />
+        {type === dataElementTypes.POLYGON && <FeatureGroup>
+            <EditControl
+                position="topright"
+                onEdited={onMapPolygonEdited}
+                onCreated={onMapPolygonCreated}
+                onDeleted={onMapPolygonDelete}
+                draw={{
+                    rectangle: false,
+                    polyline: false,
+                    circle: false,
+                    marker: false,
+                    circlemarker: false,
+                }}
+                edit={{
+                    remove: false,
+                }}
+            />
+        </FeatureGroup>}
+        {type === dataElementTypes.COORDINATE && position && <Marker position={position} />}
+    </Map>);
+
+    const getTitle = () => {
+        switch (type) {
+        case dataElementTypes.COORDINATE:
+            return i18n.t('coordinates');
+        case dataElementTypes.POLYGON:
+            return i18n.t('area');
+        default:
+            log.error(`${type} is not handled`);
+            return '';
+        }
+    };
+
+    const renderActions = () => (<ButtonStrip end>
+        <Button
+            onClick={() => {
+                setOpen(false);
+            }}
+            secondary
+        >
+            {i18n.t('Cancel')}
+        </Button>
+        <Button
+            onClick={() => {
+                if (position ?? coordinates) {
+                    const convertedCoordinates = convertToServerCoordinates(position ?? coordinates, type);
+                    // $FlowFixMe
+                    onSetCoordinates(convertedCoordinates);
+                    setOpen(false);
+                }
+            }}
+            primary
+        >
+            {`${i18n.t('Set')} ${getTitle()}`}
+        </Button>
+    </ButtonStrip>);
+
+    return (
+        <Modal
+            hide={!isOpen}
+            large
+        >
+            <ModalTitle>
```

Comment:
For enrollment without geometry, when I click the "Enrollment actions" button, an error appears in the console.
`Failed prop type: Invalid prop children of type object supplied to ModalTitle, expected string. in ModalTitle (created by MapCoordinatesModalPlain)`
Can you have a look?

<img width="1298" alt="Screenshot 2023-01-19 at 11 14 04" src="https://user-images.githubusercontent.com/6817073/213428313-49738e11-205c-4d43-8475-a1f9b470d62a.png">

Thank you!